examples/tutorials/physics_unit_tests.py

```diff
@@ -0,0 +1,312 @@
+    # place our agent in the scene
+    agent_state = habitat_sim.AgentState()
+    agent_state.position = position
+    agent_state.rotation = orientation
+    agent = sim.initialize_agent(0, agent_state)
+    return agent.scene_node.transformation_matrix()
+
+
+def make_configuration():
+    # simulator configuration
+    backend_cfg = habitat_sim.SimulatorConfiguration()
+    backend_cfg.scene_id = "NONE"
+
+    backend_cfg.enable_physics = True
+
+    # sensor configurations
+    # Note: all sensors must have the same resolution
+    # setup 2 rgb sensors for 1st and 3rd person views
+    camera_resolution = [544, 720]
+    sensors = {
+        "rgba_camera_1stperson": {
+            "sensor_type": habitat_sim.SensorType.COLOR,
+            "resolution": camera_resolution,
+            "position": [0.0, 0.6, 0.0],
+            "orientation": [0.0, 0.0, 0.0],
+        },
+    }
+
+    sensor_specs = []
+    for sensor_uuid, sensor_params in sensors.items():
+        sensor_spec = habitat_sim.SensorSpec()
+        sensor_spec.uuid = sensor_uuid
+        sensor_spec.sensor_type = sensor_params["sensor_type"]
+        sensor_spec.resolution = sensor_params["resolution"]
+        sensor_spec.position = sensor_params["position"]
+        sensor_spec.orientation = sensor_params["orientation"]
+        sensor_specs.append(sensor_spec)
+
+    # agent configuration
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+
+    return habitat_sim.Configuration(backend_cfg, [agent_cfg])
+
+
+def simulate(sim, dt=1.0, get_frames=True):
+    # simulate dt seconds at 60Hz to the nearest fixed timestep
+    # print("Simulating " + str(dt) + " world seconds.")
+    observations = []
+    physics_step_times = []
+    graphics_render_times = []
+    collisions = []
+
+    start_time = sim.get_world_time()
+    while sim.get_world_time() < start_time + dt:
+        st = time.time()
+        sim.step_physics(1.0 / 60.0)
+        mt = time.time()
+        if get_frames:
+            observations.append(sim.get_sensor_observations())
+        et = time.time()
+        physics_step_times.append(mt - st)
+        graphics_render_times.append(et - mt)
+        collisions.append(sim.get_num_active_contact_points())
+
+    return observations, physics_step_times, graphics_render_times, collisions
+
+
+# Define each test function & store in list
+
+
+def bowl_drop_test(
+    objects,
+    num_objects=30,
+    object_scale=0.5,
+    object_speed=2,
+    post_throw_settling_time=5,
+):  # take in parameters here
+    """Drops a specified number of objects into a bowl/box and returns metrics including the time to simulate each frame, render each frame, and the number of collisions each frame. """
+    # start with bowl being bounding boxes
+
+    cfg = make_configuration()
+    try:  # Got to make initialization idiot proof
+        sim.close()
+    except NameError:
+        pass
+
+    sim = habitat_sim.Simulator(cfg)
```

Comment:
If you use the ```with  habitat_sim.Simulator(cfg) as sim:``` construct for all these tests you won't have to worry about closing the simulator.